doc/source/conf.py

```diff
@@ -101,12 +102,22 @@
 # further.  For a list of options available for each theme, see the
 # documentation.
 #
-# html_theme_options = {}
+html_theme_options = {
```

Comment:
I think those change the looks of documentation quite a lot. I'm not saying it's wrong, but lets review it together after merging.